src/content/peerconnection/audio/index.html

```diff
@@ -27,55 +26,67 @@
   <link href="//fonts.googleapis.com/css?family=Inconsolata" rel="stylesheet" type="text/css">
   <link rel="stylesheet" href="../../../css/main.css" />
   <link rel="stylesheet" href="css/main.css" />
-
 </head>
 
 <body>
-
   <div id="container">
-
-    <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC samples</a> <span>Peer connection: audio only</span></h1>
-
-    <div id="audio">
-      <div>
-        <div class="label">Local audio:</div><audio id="audio1" autoplay controls muted></audio>
-      </div>
-      <div>
-        <div class="label">Remote audio:</div><audio id="audio2" autoplay controls></audio>
-      </div>
+  <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC samples</a> <span>Peer connection: audio only</span></h1>
```

Comment:
Indent is incorrect, should belong to the container div. Applies throughout the file.